### ISSUE TITLE
Script load ever and do not respect a condition for a trigger manual

### ISSUE THREAD

**Issue by Met96** [nuxt/scripts#307]
### 📚 What are you trying to do?

I make the follow code for run the script only in a specific countries, but it will be loaded ever without respect the condition
The follow code is inside a composable called in the app.vue of nuxt project.

Can you help me ?

### 🔍 What have you tried?

const notLoadCookieBanner = ['en-us'] 
const countryLocale = ref('es-es')
const {
    onLoaded,
    load: loadCookie,
  } = useScript<OneTrust>(
    {
      src: 'https://cdn.cookielaw.org/scripttemplates/otSDKStub.js',
      'data-domain-script': '....',
      defer: true,
    },
    {
      trigger: 'manual',
      use() {
        return window.OneTrust
      },
    }
  )

  if (
    countryLocale.value &&
    notLoadCookieBanner.findIndex(
      countryLocaleBanner => countryLocaleBanner === countryLocale.value
    ) === -1
  ) {
    loadCookie()
  }

**Comment by harlan-zw** [comment]
Hi, can you please make a minimal reproduction? 

You can use the starter here: https://stackblitz.com/edit/nuxt-starter-pkwfkx?file=pages%2Findex.vue

**Comment by Met96** [comment]
I tried it with stackblitz but it's running correctly, so i think it was a problem with Netlify or cache system

**Comment by harlan-zw** [comment]
I need a way to replicate it to fix it unfortunately

**Comment by Met96** [comment]
Yeah i know, so in next days i'll send you more updates to test it
